modules/p2/ting_carousel/ting_carousel.module

```diff
@@ -74,12 +74,8 @@ function ting_carousel_preprocess_ting_object(&$variables, $hook) {
   if (isset($entity_uri)) {
     $entity_link = url($entity_uri['path'], array('absolute' => TRUE));
     $variables['content']['ting_cover']['#prefix'] = '<div class="contentbox"><a href="' . $entity_link . '">';
```

Comment:
why not use a render array for the link, and div container, rather than a #prefix + #suffix?
I have an example at: https://github.com/JrgenGNielsen/drupal7_examples/tree/master/render_arrays/drupal7_examples_div_container